cypress/e2e/x-button.cy.js

```diff
@@ -0,0 +1,15 @@
+describe('test for the x button that clears test on editor', () => {
+    beforeEach(() => {
+        cy.visit('http://localhost:4200/'); 
+    })
+    
+it('will click on the "x" button and check if the text is clear', () => {
+
+    cy.get("#editor > p > .typo").should("not.exist");
+    cy.get("#editor").type("test per butonin x");
+    cy.get('i.bi-x').click();
+    cy.get("#editor").contains("test per butonin x").should("not.exist");
+
+});
+
+});
```

Comment:
Can you also test here that the delete button is not shown if there's no text in the editor? Perhaps you can check on line 7 and 12 whether it is shown.